java/build.xml

```diff
@@ -132,6 +132,11 @@
 
   <target name="init-install" depends="set-default-config-dir">
     <property name="installbuild" value="true" />
+    <property name="customserverdir" value="/srv" />
+    <property name="sharedserverdir" value="/srv" />
+    <replace file="code/src/com/suse/manager/webui/services/SaltConstants.java" token="@SHAREDSERVDIR@" value="${sharedserverdir}" />
```

Comment:
Are these needed at build time? 
Instead, can we do it all in java, have the default values for it and some domain method in class `ConfigDefaults`?

Reply:
The idea behind this was to define the values in the SPEC file/central place and remove the number of places to put the values.
Apart from that, I have no feelings/specific attachments on where/how to put logic.
`ConfigDefaults` sounds find but as I am not a Java expert, I would need some assistance/triple reviews to get this sorted. Let me get started on this.

Reply:
Sure I can review it. Thanks for the contribution